PartsAppTests/ExcelHelperTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace PartsAppTests
+{
+    [TestClass]
+    public class ExcelHelperTests
+    {
+        private readonly Func<string, string> _method = PartsApp.ExcelHelper.OperationsExcelHelper.GetValidExcelBookTitle;
```

Comment:
don't do like this. Even if you use the same method in multiple places. If this is exactly the method you're trying to test then just use it directly as many times as many tests you have

Reply:
corrected